zulipterminal/config/themes.py

```diff
@@ -95,6 +95,7 @@
 THEME_ALIASES = {
     "default": "zt_dark",
     "gruvbox": "gruvbox_dark",
+    "nord": "nord_dark",
```

Comment:
This section was intended for historical aliases; I'd prefer we avoid supporting "nord" as a theme name for now, which is what this line would do.